### PR TITLE
Refactoring code.

### PR DESCRIPTION
**Pull request by Manuelraa**
Should all still work

### CODE REVIEW

**Comment by Manuelraa** [comment]
I will change some more things. Found some unnecessary code

**Comment by RednibCoding** [comment]
Traceback (most recent call last):
  File "server.py", line 110, in <module>
    main()
  File "server.py", line 106, in main
    RpiServer()()
  File "server.py", line 39, in __init__
    self.setup_config()
  File "server.py", line 88, in setup_config
    self.cfg.cron = self.cfg['Cron']
  File "/usr/lib/python3.5/configparser.py", line 956, in __getitem__
    raise KeyError(key)
KeyError: 'Cron'

File: rpi_server/rpi_server.py [hunk 1]
```diff
@@ -1,101 +1,110 @@
...
+    def __del__(self):
+        '''Gets always executet if the instance is destroyed.
+        Also if an Exeption happens
+        '''
+        gpio.cleanup()
+
+    def setup_config(self):
+        self.cfg = ConfigParser()
+        self.cfg.read("config.ini")
+        self.cfg.cron = self.cfg['Cron']
```
Comment (RednibCoding) [inline rpi_server/rpi_server.py:87]: cfg hat kein attribut cron

File: rpi_server/rpi_server.py [hunk 2]
```diff
@@ -1,101 +1,110 @@
...
+    '''Gets only called when run in an python shell, Ignoring imports'''
+
+    # Setup GPIO'S
+    gpio.setwarnings(False)
+    gpio.setmode(gpio.BCM)
+
+    for pin in OPINS:
+        if pin:
+            gpio.setup(pin, gpio.OUT)
+    RpiServer()()
```
Comment (RednibCoding) [inline rpi_server/rpi_server.py]: ?

File: rpi_server/rpi_server.py (hunk 1 above)
Comment (Manuelraa) [reply rpi_server/rpi_server.py:87]: Bei mir ging es auch das setzten durch den Socket. Ich schau nochmal drüber

**Comment by Manuelraa** [comment]
Okay is ready to be merged now.

**Review by RednibCoding — APPROVED** [review]

**Pull request merged**